docker-compose.yml

```diff
@@ -57,6 +67,12 @@ services:
       default:
         aliases:
          - wdqs.svc
+    environment:
+      - WIKIBASE_HOST=wikibase.svc
+      - WDQS_HOST=wdqs.svc
+      - WDQS_PORT=9999
+    expose:
+      - 9999
```

Comment:
port 9999 being exposed again?

Reply:
As above, because its now removed from wdqs/0.2.5/Dockerfile
and wdqs/0.3.0/Dockerfile